### PR TITLE
added clearSavedState method

### PR DESCRIPTION
**Pull request by AlexVanderbist**
Something like this for #3?

### CODE REVIEW

File: __tests__/helpers/LocalStorageMock.js [hunk 1]
```diff
@@ -4,14 +4,17 @@ const localStorageMock = (function () {
 
     return {
         getItem(key) {
-            return store[key];
+            return store[key] || null;
```
Comment (sebastiandedeyne) [inline __tests__/helpers/LocalStorageMock.js:7]: Why the sudden `null`? `Storage` instances return undefined when a key doesn't exist iirc.

**Review by sebastiandedeyne — CHANGES_REQUESTED** [review]

File: __tests__/helpers/LocalStorageMock.js (hunk 1 above)
Comment (AlexVanderbist) [reply __tests__/helpers/LocalStorageMock.js:7]: According to https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem they should return `null` if the key doesn't exist (tested in devtools as well)

Comment (sebastiandedeyne) [reply __tests__/helpers/LocalStorageMock.js:7]: Oh, TIL, looks good then! :)

**Comment by sebastiandedeyne** [comment]
Cool, thanks!

**Pull request merged**